### PR TITLE
Remove context.state

### PR DESCRIPTION
**Pull request by nonumpa**
1. Use different SOURCE_PREFIX to handle first submission article and not-yet replied article logic
  - Prefix [1️⃣ I got the message from] for first submission article
  - Prefix [ℹ️ I got the message from] for not-yet replied article
  - When `createAskArticleSubmissionConsentReply`, send a liff.source url with `article_submission=true`
 - Create new session after submitting article (make article submission button expire)

2. Remove context.state

### CODE REVIEW

File: src/webhook/handlers/__tests__/askingArticleSubmissionConsent.test.js [hunk 1]
```diff
@@ -34,10 +34,9 @@ it('should block incorrect prefix', async () => {
       searchedText: 'Some text forwarded by the user',
       foundArticleIds: [],
     },
-    state: 'ASKING_ARTICLE_SUBMISSION_CONSENT',
     event: {
       type: 'message',
-      input: REASON_PREFIX + 'foo', // Wrong prefix
+      input: SOURCE_PREFIX_NOT_YET_REPLIED + 'foo', // Wrong prefix
```
Comment (MrOrz) [inline src/webhook/handlers/__tests__/askingArticleSubmissionConsent.test.js:39]: I am wondering if it is possible for `should block incorrect prefix` to happen under the current action handling logic, since `SOURCE_PREFIX_NOT_YET_REPLIED` will be redirected to other state for now.

If not, I think we can remove the logic that emits `ManipulationError` together.

File: src/webhook/handlers/askingArticleSubmissionConsent.js [hunk 2]
```diff
@@ -59,6 +62,9 @@ export default async function askingArticleSubmissionConsent(params) {
       CreateArticle.id
     );
 
+    // Create new session, make article submission button expire after submitting
+    data.sessionId = Date.now();
```
Comment (MrOrz) [inline src/webhook/handlers/askingArticleSubmissionConsent.js:66]: This is brilliant 👍 

Cannot think of edge cases here. Let's test this in Wednesday.

**Review by MrOrz — APPROVED** [review]
LGTM! This is a stepping stone PR that leads us away from state diagram towards future models.

Thanks for the fix.

**Review by MrOrz — APPROVED** [review]
LGTM! Let's test it tonight 🚀
